scaleapi/api.py

```diff
@@ -1,27 +1,20 @@
 import platform
 import urllib.parse
-
 import requests
 from requests.adapters import HTTPAdapter, Response, Retry
-
 from ._version import __package_name__, __version__
 from .exceptions import ExceptionMap, ScaleException
-
 SCALE_API_BASE_URL_V1 = "https://api.scale.com/v1"
-
 # Parameters for HTTP retry
 HTTP_TOTAL_RETRIES = 3  # Number of total retries
 HTTP_RETRY_BACKOFF_FACTOR = 2  # Wait 1, 2, 4 seconds between retries
 HTTP_STATUS_FORCE_LIST = [408, 429] + list(range(500, 531))
 HTTP_RETRY_ALLOWED_METHODS = frozenset({"GET", "POST", "DELETE"})
-
-
 class Api:
     """Internal Api reference for handling http operations"""
-
     def __init__(
         self,
-        api_key,
+        api_key, 
```

Comment:
nit: I feel like removing those new lines makes code a bit hard to read, wdyt?

Reply:
Yea, fixing the local formatting issue.